demo/services/mockWorkspaceFlowRunsApi.ts

```diff
@@ -26,11 +27,27 @@ export class MockWorkspaceFlowRunsApi extends MockApi implements IWorkspaceFlowR
   }
 
   public getFlowRunsHistory(filter: FlowRunsHistoryFilter): Promise<RunHistory[]> {
-    if (Object.keys(filter).length) {
-      console.warn('MockWorkspaceFlowRunsApi has not implemented the filter argument of the getFlowRunsCount method')
+    let start = mapper.map('string', filter.history_start, 'Date')
+    const end = mapper.map('string', filter.history_end, 'Date')
+    const runHistory: RunHistory[] = []
+
+    while (dateFunctions.isBefore(start, end)) {
```

Comment:
Does `isBefore` also guarantee that passed arguments are both defined? I would assume so

Reply:
Well the [underlying date-fns method](https://date-fns.org/v2.29.3/docs/isBefore) does not accept `null` or `undefined`, so I imagine there would be an error. I would probably be more concerned with handling that if this weren't in the demo

Reply:
Ah yeah fair enough